.expeditor/config.yml

```diff
@@ -39,26 +40,28 @@ changelog:
     - "Type: Enhancement": "Enhancements"
     - "Type: Bug": "Bug Fixes"
 
-merge_actions:
-  - built_in:bump_version:
-      ignore_labels:
-        - "Version: Skip Bump"
-        - "Expeditor: Skip All"
-  - bash:.expeditor/update_version.sh:
-      only_if: built_in:bump_version
-  - built_in:update_changelog:
-      ignore_labels:
-        - "Changelog: Skip Update"
-        - "Expeditor: Skip All"
-  - built_in:trigger_omnibus_release_build:
-      ignore_labels:
-        - "Omnibus: Skip Build"
-        - "Expeditor: Skip All"
-      only_if: built_in:bump_version
-  - built_in:trigger_habitat_package_build:
-      ignore_labels:
-        - "Habitat: Skip Build"
-        - "Expeditor: Skip All"
+subscriptions:
+  - workload: pull_request_merged:{{github_repo}}:{{release_branch}}:*
+    actions:
+     - built_in:bump_version:
+        ignore_labels:
+         - "Version: Skip Bump"
+         - "Expeditor: Skip All"
+     - bash:.expeditor/update_version.sh:
+        only_if: built_in:bump_version
+     - built_in:update_changelog:
+        ignore_labels:
+         - "Changelog: Skip Update"
+         - "Expeditor: Skip All"
+     - built_in:trigger_omnibus_release_build:
+        ignore_labels:
+         - "Omnibus: Skip Build"
+         - "Expeditor: Skip All"
+        only_if: built_in:bump_version
+     - built_in:trigger_habitat_package_build:
+        ignore_labels:
+         - "Habitat: Skip Build"
+         - "Expeditor: Skip All"
 
 artifact_actions:
```

Comment:
This needs to be converted into subscriptions. 

`- workload: artifact_published:unstable:inspec:1*`
`- workload: artifact_published:current:inspec:1*`
`- workload: artifact_published:stable:inspec:1*`